pkg/target/target.go

```diff
@@ -88,14 +94,68 @@ func (h *K8sValidationTarget) HandleReview(obj interface{}) (bool, interface{},
 		return true, data, nil
 	case *admissionv1beta1.AdmissionRequest:
 		return true, data, nil
-	case SideloadNamespace:
-		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
-	case *SideloadNamespace:
-		return true, &augmentedReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case AugmentedReview:
+		return true, &gkReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case *AugmentedReview:
+		return true, &gkReview{AdmissionRequest: data.AdmissionRequest, Unstable: &unstable{Namespace: data.Namespace}}, nil
+	case AugmentedUnstructured:
+		admissionRequest, err := augmentedUnstructuredToAdmissionRequest(data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
+	case *AugmentedUnstructured:
+		admissionRequest, err := augmentedUnstructuredToAdmissionRequest(*data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
+	case unstructured.Unstructured:
+		admissionRequest, err := unstructuredToAdmissionRequest(data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
+	case *unstructured.Unstructured:
+		admissionRequest, err := unstructuredToAdmissionRequest(*data)
+		if err != nil {
+			return false, nil, err
+		}
+		return true, admissionRequest, nil
 	}
 	return false, nil, nil
 }
 
+func augmentedUnstructuredToAdmissionRequest(obj AugmentedUnstructured) (admissionv1beta1.AdmissionRequest, error) {
+	req, err := unstructuredToAdmissionRequest(obj.Object)
+	if err != nil {
+		return req, err
+	}
+	req.Namespace = obj.Namespace.Namespace
```

Comment:
here is where we would be defining Unstable.Namespace

Reply:
:+1: